integrations/census/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Census",
+  "version": "1.0.0",
+  "overview": "Use Census to sync metrics and metric events from your data warehouse into LaunchDarkly.",
+  "description": "Your data warehouse is the source of truth for how users interact with your product. Now you can use it to supercharge your LaunchDarkly experiments by syncing metrics and metric events.",
+  "author": "Census",
+  "supportEmail": "support@getcensus.com",
+  "links": {
+    "site": "https://www.getcensus.com",
```

Comment:
```suggestion
    "site": "https://www.getcensus.com",
    "launchdarklyDocs": "https://docs.getcensus.com/destinations/launchdarkly",
```

Reply:
@ncarchedi If we can get this suggestion added in, I'll pull this data locally and test to verify then we should be good to merge.